docs/DeveloperGuide.md

```diff
@@ -19,6 +21,32 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Use Cases**
+
+### Adding a New Contact
+#### **Actors:** Teaching Assistant (TA)
+#### **Preconditions:**
+- The TA has the application installed.
+- The TA is on the main screen of the address book.
+
+#### **Steps:**
+1. The TA adds the contact details
+2. The TA sees the newly added contact in the address book.
+
+### Editing a Contact
+#### **Actors:** Teaching Assistant (TA)
+#### **Preconditions:**
+- The TA has an existing contact that needs to be updated.
+
+#### **Steps:**
+1. The TA searches for and selects the contact.
+1a. If the contact does not exist, perform **Adding a New Contact**.
```

Comment:
The formatting is incorrect here -- step 1a appears on the same line as step 1 when rendered.